CitizenLabDotCo/citizenlab
# CL-979: Make height field number and add iframeDescription

## Checklist

- [ ] Added entry to change log

## How to test
- Navigate to any project description with the content builder turned on
- Add an embed component
- Check to see that the height only accepts numbers
- Check to see that there is a description there

## Links

- [CL-979](https://citizenlab.atlassian.net/browse/CL-979)

## Screenshots
<img width="1436" alt="image" src="https://user-images.githubusercontent.com/12659000/173854939-710bd0ee-aca1-4fdb-b527-b0f01c0cc017.png">


## How urgent is a code review?

End of day Thursday
